collect_app/src/main/java/org/odk/collect/android/tasks/InstanceUploader.java

```diff
@@ -153,6 +167,17 @@ public void setUploaderListener(InstanceUploaderListener sl) {
         }
     }
 
+    private void deferSubmissionStatus(Iterator<String> instanceIds) {
+
```

Comment:
Please remove this blank line. Is this being added by Android Studio? Can you turn it off?

Is there a particular reason to pass in the iterator? I think it would be clearer to call the set `instanceIds` and pass that in. Then the semantics are clear -- you're deferring submission status on the instanceIDs. Now the parameter name suggests that's what you want to do but actually you're passing in an iterator on the collection. Using a for each loop also makes things easier to read (that's why it's provided as syntactic sugar on top of iterators!).

Reply:
Sure! Will do! I will check if Android Studio is actually adding it.